bridge/client/app/dashboard-legacy/dashboard-legacy.component.spec.ts

```diff
@@ -49,7 +51,26 @@ describe('DashboardLegacyComponent', () => {
     expect(loadProjectSpy).toHaveBeenCalled();
   });
 
+  it('should load a maximum of 5 sequences per project', (done) => {
```

Comment:
@renepanzar @Kirdock @ermin-muratovic do you have a better solution to test complex RXJS observables?